fe/src/components/GameBoard/constants.ts

```diff
@@ -93,3 +91,35 @@ export const changeDirection = (direction: DirectionType) => {
       return 'top';
   }
 };
+
+export const CELL_COORDINATES: { [key: number]: { x: number; y: number } } = {
+  0: { x: 0, y: 0 },
+  1: { x: 0, y: -6 },
+  2: { x: 0, y: -12 },
+  3: { x: 0, y: -18 },
+  4: { x: 0, y: -24 },
+  5: { x: 0, y: -30 },
+  6: { x: 0, y: -36 },
+  7: { x: 6, y: -36 },
+  8: { x: 12, y: -36 },
+  9: { x: 18, y: -36 },
+  10: { x: 24, y: -36 },
+  11: { x: 30, y: -36 },
+  12: { x: 36, y: -36 },
+  13: { x: 36, y: -30 },
+  14: { x: 36, y: -24 },
+  15: { x: 36, y: -18 },
+  16: { x: 36, y: -12 },
+  17: { x: 36, y: -6 },
+  18: { x: 36, y: 0 },
+  19: { x: 30, y: 0 },
+  20: { x: 24, y: 0 },
+  21: { x: 18, y: 0 },
+  22: { x: 12, y: 0 },
+  23: { x: 6, y: 0 },
+};
```

Comment:
타입을 명시하신 이유가 있을까요??

Reply:
useTeleportToken() 훅에 내부
```ts
const getCellCoordinates = (location: number) => CELL_COORDINATES[location];
```
 요 부분에서 사용할 때 다음과 같은 타입에러가 나서 추가한 부분입니다. 
`Element implicitly has an 'any' type because expression of type 'number' can't be used to index type ts(7053)`

`CELL_COORDINATES` 객체의 키값이 number 타입임을 명시해주니까 에러가 없어지더라구요.